src/EntryPoint/EntryPoints.ts

```diff
@@ -1,10 +1,12 @@
+export interface EntryPointFilter {
```

Comment:
Having `limit` may imply that it is more than just filter; it has a broader scope, let's rename:

```suggestion
export interface EntryPointsListOptions {
```